charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -245,6 +245,8 @@ controller:
   deploymentAnnotations: {}
   podAnnotations: {}
   podLabels: {}
+  # Warning: Disabling PodDisruptionBudget may lead to stateful workloads not starting due to lack of controller pods
+  podDisruptionBudget: true
```

Comment:
Would it be worth making this `controller.podDisruptionBudget.enabled`, in case we want to later add something like `contoller.podDisruptionBudget.minAvailable`?

Otherwise lgtm